component/src/main/java/org/wso2/carbon/identity/authenticator/bitly/BitlyAuthenticator.java

```diff
@@ -263,4 +273,43 @@ protected Map<ClaimMapping, String> getSubjectAttributes(OAuthClientResponse tok
         }
         return claims;
     }
+
+    @Override
+    protected String sendRequest(String url, String accessToken) throws IOException {
+
+        if (log.isDebugEnabled()) {
+            log.debug("URL: " + url);
+        }
+
+        if (url == null) {
+            return StringUtils.EMPTY;
+        }
+
+        StringBuilder builder = new StringBuilder();
+        BufferedReader reader = null;
+
+        try {
+            URL obj = new URL(url);
+            HttpURLConnection urlConnection = (HttpURLConnection) obj.openConnection();
+            urlConnection.setRequestMethod(GET_METHOD);
+            urlConnection.setRequestProperty(AUTHORIZATION_HEADER, BEARER_HEADER + " " + accessToken);
+            urlConnection.setRequestProperty(ACCEPT_HEADER, APPLICATION_JSON_HEADER);
+            reader = new BufferedReader(new InputStreamReader(urlConnection.getInputStream()));
+            String inputLine = reader.readLine();
+
+            while (inputLine != null) {
+                builder.append(inputLine).append("\n");
+                inputLine = reader.readLine();
+            }
+        } finally {
+            if (reader != null) {
+                reader.close();
+            }
+        }
+
+        if (log.isDebugEnabled() && IdentityUtil.isTokenLoggable(IdentityConstants.IdentityTokens.USER_ID_TOKEN)) {
+            log.debug("Bitly user info response: " + builder.toString());
```

Comment:
We are not printing the token then why are we checking isTokenLoggable?

Reply:
Yea, we can remove this condition

Reply:
Fixed